src/main/java/io/swagger/api/media/MediaApiController.java

```diff
@@ -31,16 +33,22 @@
 
     private final HttpServletRequest request;
 
+    private final MediaService mediaService;
+
     @org.springframework.beans.factory.annotation.Autowired
-    public MediaApiController(ObjectMapper objectMapper, HttpServletRequest request) {
+    public MediaApiController(ObjectMapper objectMapper, HttpServletRequest request, MediaService mediaService) {
         this.objectMapper = objectMapper;
         this.request = request;
+        this.mediaService = mediaService;
     }
 
     public ResponseEntity<SearchData> mediaGet(@NotNull @Parameter(in = ParameterIn.QUERY, description = "" ,required=true,schema=@Schema()) @Valid @RequestParam(value = "query", required = true) String query) {
         String accept = request.getHeader("Accept");
         if (accept != null && accept.contains("application/json")) {
             try {
+                // tt0461770 is the only existing test data (for now...)
+                MediaInfo mediaInfo = mediaService.GetMediaById("tt0461770");
+                System.out.println(mediaInfo);
```

Comment:
This is all I did to verify that data is being pulled out of the database correctly. MediaInfo is the database object and does not yet have all the necessary items yet.